### PR TITLE
U4-6236 - umbraco 7 macro parameters saving "null"

### PR DESCRIPTION
**Pull request by jozsurf**
Text editor should have a value of blank instead of "null" if it doesn't
contain anything

### CODE REVIEW

**Comment by jozsurf** [comment]
This is the first time I'm doing a PR for Umbraco - any ideas why CI failed?

**Comment by Shazwazza** [comment]
Just click on the 'Details' button on the right of the AppVeyor thing and it will tell you. However, in some cases its due to connectivity issues and some other things. So not sure if it's your code or just the integration.

**Comment by rjbullock** [comment]
REALLY need this fixed myself.

File: src/Umbraco.Web.UI.Client/src/common/directives/umbeditor.directive.js [hunk 1]
```diff
@@ -28,6 +28,10 @@ angular.module("umbraco.directives")
                 }
 
                 scope.propertyEditorView = umbPropEditorHelper.getViewPath(scope.model.view, scope.isPreValue);
+				
+				 if(scope.model.view == 'textbox'){
```
Comment (Shazwazza) [inline src/Umbraco.Web.UI.Client/src/common/directives/umbeditor.directive.js]: This is ultra hacky, we need to come up with a better solution than this, i'll investigate soon.

**Comment by Shazwazza** [comment]
This is fixed correctly in rev: f4bdcae411ab8cadaf534bc55f66056e8bf29079
closing this PR.

**Pull request closed without merging**